packages/grafana-ui/src/components/Table/TableNG/TableNG.tsx

```diff
@@ -265,6 +265,9 @@ export function TableNG(props: TableNGProps) {
         cellRootRenderers: {},
       };
 
+      let lastRowIdx = -1;
+      let _rowHeight = 0;
```

Comment:
oops these weren't in the correct scope, so we remeasured the row as many times as we had columns.